client/src/common/components/CollectionView.jsx

```diff
@@ -51,48 +52,71 @@ class GridList extends PureComponent {
     history.push(`/${route}/${itemId}`);
   };
 
+  renderAddNew = () => {
+    const { onAddNew } = this.props;
+
+    return (
+      <li className="collection__item">
+        <button className="collection__button" onClick={onAddNew} type="button">
+          <CardPlus />
+        </button>
+      </li>
+    );
+  };
+
+  renderListMode = () => {
+    const { color, items, route } = this.props;
+
+    return _map(items, item => (
+      <li className="collection__item" key={item._id}>
+        <ListModeItem
+          color={color}
+          item={item}
+          onCardClick={this.handleCardClick(route, item._id)}
+          onFavClick={this.handleFavClick(item._id, item.isFavourite)}
+          route={route}
+        />
+      </li>
+    ));
+  };
+
+  renderTilesMode = () => {
```

Comment:
renderAsTiles 

- renderTilesMode means that you would like to render some "Mode" as element, component or switch button or something else. The mode is like an option for me that I can choose by interaction.